From 63de12a2a94122562317a24c566dda4f304e5a52 Mon Sep 17 00:00:00 2001
From: Kunhui-Li <kunhuix.li@intel.com>
Date: Thu, 5 Aug 2021 14:46:26 +0800
Subject: [PATCH 6/6] config_tools: update launch xml for only one NVME

Update launch xml to use image to launch hard rt vm since
we changed the platforms from two disk to only one NVME disk.

Tracked-On: #6315
Signed-off-by: Kunhui-Li <kunhuix.li@intel.com>
---
 misc/config_tools/data/adl-s-crb/industry_launch_2uos.xml | 6 +++---
 misc/config_tools/data/adl-s-crb/industry_launch_6uos.xml | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/misc/config_tools/data/adl-s-crb/industry_launch_2uos.xml b/misc/config_tools/data/adl-s-crb/industry_launch_2uos.xml
index 4ab01d47c..a4958b50a 100644
--- a/misc/config_tools/data/adl-s-crb/industry_launch_2uos.xml
+++ b/misc/config_tools/data/adl-s-crb/industry_launch_2uos.xml
@@ -75,14 +75,14 @@
 		<bluetooth desc="vm bluetooth"></bluetooth>
 		<sd_card desc="vm sd card device"></sd_card>
 		<ethernet desc="vm ethernet device"></ethernet>
-		<sata desc="vm sata device">00:17.0 SATA controller: Intel Corporation Device a0d3 (rev 20)</sata>
+		<sata desc="vm sata device"></sata>
 		<nvme desc="vm nvme device"></nvme>
 	</passthrough_devices>
 
 	<virtio_devices>
-		<network desc="virtio network devices setting. Input format: tap_name,[vhost],[mac=XX:XX:XX:XX:XX:XX]."></network>
+		<network desc="virtio network devices setting. Input format: tap_name,[vhost],[mac=XX:XX:XX:XX:XX:XX].">RT</network>
 		<input desc="virtio input device"></input>
-		<block desc="virtio block device setting. format: [blk partition:][img path] e.g.: /dev/sda3:./a/b.img"></block>
+		<block desc="virtio block device setting. format: [blk partition:][img path] e.g.: /dev/sda3:./a/b.img">./core-image-weston-intel-corei7-64.wic</block>
 		<console desc="virtio console device,input format: [@]stdio|tty|pty|sock:portname[=portpath][,[@]stdio|tty|pty:portname[=portpath]]">@stdio:stdio_port</console>
 	</virtio_devices>
     </uos>
diff --git a/misc/config_tools/data/adl-s-crb/industry_launch_6uos.xml b/misc/config_tools/data/adl-s-crb/industry_launch_6uos.xml
index 39577ac9f..d7a741bc4 100644
--- a/misc/config_tools/data/adl-s-crb/industry_launch_6uos.xml
+++ b/misc/config_tools/data/adl-s-crb/industry_launch_6uos.xml
@@ -75,14 +75,14 @@
 		<bluetooth desc="vm bluetooth"></bluetooth>
 		<sd_card desc="vm sd card device"></sd_card>
 		<ethernet desc="vm ethernet device"></ethernet>
-		<sata desc="vm sata device">00:17.0 SATA controller: Intel Corporation Device a0d3 (rev 20)</sata>
+		<sata desc="vm sata device"></sata>
 		<nvme desc="vm nvme device"></nvme>
 	</passthrough_devices>
 
 	<virtio_devices>
-		<network desc="virtio network devices setting. Input format: tap_name,[vhost],[mac=XX:XX:XX:XX:XX:XX]."></network>
+		<network desc="virtio network devices setting. Input format: tap_name,[vhost],[mac=XX:XX:XX:XX:XX:XX].">RT</network>
 		<input desc="virtio input device"></input>
-		<block desc="virtio block device setting. format: [blk partition:][img path] e.g.: /dev/sda3:./a/b.img"></block>
+		<block desc="virtio block device setting. format: [blk partition:][img path] e.g.: /dev/sda3:./a/b.img">./core-image-weston-intel-corei7-64.wic</block>
 		<console desc="virtio console device,input format: [@]stdio|tty|pty|sock:portname[=portpath][,[@]stdio|tty|pty:portname[=portpath]]">@stdio:stdio_port</console>
 	</virtio_devices>
     </uos>
-- 
2.25.1

